phpBB/styles/prosilver/template/mcp_header.html

```diff
@@ -19,31 +19,31 @@
 <div class="panel bg3">
 	<div class="inner">
 
-	<div style="width: 100%;">
+		<div style="width: 100%;">
 
-	<div id="cp-menu">
-		<div id="navigation" role="navigation">
-			<ul>
-			<!-- BEGIN l_block1 -->
-			<!-- IF l_block1.S_SELECTED -->
-				<!-- BEGIN l_block2 -->
-				<!-- IF l_block1.l_block2.S_SELECTED -->
-				<li id="active-subsection"><a href="{l_block1.l_block2.U_TITLE}"><span>{l_block1.l_block2.L_TITLE}<!-- IF l_block1.l_block2.ADD_ITEM --> ({l_block1.l_block2.ADD_ITEM})<!-- ENDIF --></span></a></li>
-				<!-- ELSE -->
-				<li><a href="{l_block1.l_block2.U_TITLE}"><span>{l_block1.l_block2.L_TITLE}<!-- IF l_block1.l_block2.ADD_ITEM --> ({l_block1.l_block2.ADD_ITEM})<!-- ENDIF --></span></a></li>
-				<!-- ENDIF -->
-				<!-- END l_block2 -->
-			<!-- ENDIF -->
-			<!-- END l_block1 -->
-			</ul>
-		</div>
-	</div>
+			<div id="cp-menu" class="cp-menu">
```

Comment:
@marc1706 @nickvergessen this is not really a vaild argument anymore as recent changes to github compound highlight on changes. You can see the spacing added as well as any other changes to the code. All indentation made is to fix inconsistent spacing that exists for cleaner codebase.